documentation/docs/student/quiz-training-mode.mdx

```diff
@@ -0,0 +1,204 @@
+
+### Answering Questions
+
+Artemis presents questions in three different formats:
+
+- **Multiple-Choice Questions**
+
+- **Drag-and-Drop Questions**
+
+- **Short Answer Questions**
+
+### Graded vs. Ungraded Questions
+
+<Image src={unratedNotification} alt={"Unrated Notification"} size={ImageSize.medium}></Image>
+
+<Callout variant={CalloutVariant.info}>
+        When an ungraded question appears, you'll be asked if you want to continue.
+        Ungraded questions don't count toward your score but help with practice.
+</Callout>
+
+---
+
+## Video Tutorial
+
+<iframe
+    src="https://live.rbg.tum.de/w/artemisintro/67223?video_only=1&t=0"
+    allowFullScreen="1"
+    frameBorder="0"
+    width="600"
+    height="400"
+></iframe>
+
+---
+
+## Tips for Effective Training
+
+1. **Practice Regularly:** Visit the training daily to benefit from spaced repetition
+2. **Work Through All Questions:** Use both graded and ungraded questions for learning
+3. **Read Feedback:** Understand why an answer was correct or incorrect
+4. **Use the Leaderboard:** Get motivated by others and compare your progress
+5. **Work Toward Higher Leagues:** Set goals for the next league level
+
+---
+
+## Frequently Asked Questions
+
+**When should I answer questions again?**
+The system displays a countdown showing when the next questions are available for review. This is based on your past performance.
+
+**Can I change my leaderboard visibility?**
+Yes, click the **"View description"** button on the overview page to adjust your settings. You can also change the settings in your user settings under Quiz Training
+
+**What happens if I answer a question incorrectly?**
+The question will be repeated at shorter intervals until you answer it correctly. You will not earn or lose points for wrong answered questions.
+
+**Do all questions count toward my score?**
+Only graded questions count toward your score and league progress. Ungraded questions are solely for practice.
+
+**I answered a graded question correctly, but did not receive points. Why?**
+If you answered the question wrong before, you will not receive points for the same question if you answer it correctly on the same day. This is to prevent users from repeatedly answering a question wrong and then correct to secure points.
+
+**What is spaced-repetition?**
+Spaced repetition is a learning technique that involves reviewing information at increasing intervals to enhance long-term retention.
+Correctly answered questions are shown less frequently, while incorrectly answered questions appear more often.
+
+<Callout variant={CalloutVariant.tip}>
+        Use Quiz Training regularly to reinforce your knowledge long-term.
+        The spaced-repetition method is scientifically proven to be one of the most effective learning methods!
```

Comment:
"H. Ebbinghaus, “Memory: A Contribution to Experimental Psychology" is that a fitting source here?